app/views/user_mailer/shared/_in_person_ready_to_verify.html.erb

```diff
@@ -275,6 +275,21 @@
   </div>
 <% end %>
 
+<%# alert %>
```

Comment:
I checked this in user mailer preview with in_person_proofing_post_office_closed_alert_enabled set to both true and false. Both looked good.